### DISCUSSION TITLE
Case fans plugged into fan hub are greyed out

### DISCUSSION THREAD

**Discussion by arcline11** [Rem0o/FanControl.Releases#897] [Q&A] [answered]
Just completed a build in a Fractal Torrent case, which has a fan hub. I plugged all six case fans into the fan hub, then the fan hub into a single Sys_Fan header on a Gigabyte Aorus Pro motherboard. Here's a link to a pic of [my Fan Control open window](https://i.imgur.com/EocEhI2.png). What must I do so that Fan Control is controlling my case fans? I'm fine with all case fans moving up and down together. Also, total noob question, for Fan Control settings to control fans in-game, must Fan Control be open while game is running? Sorry, one more: I use MSI Afterburner to monitor in-game sensors, it looks to me like Fan Control settings trump the MSI fan setting, but I want to verify. 

Many thanks if you have the patience to answer my questions. Love your application. Just amazing. Thank you.

**Comment by Rem0o** [comment, marked as answer]
All case fans connected to that 1 hub should be 1 control card in fancontrol corresponding to that header.

FanControl must be open to operate.

FanControl can override Afterburner if you activate the the control card corresponding to your gpu.  Use one or the other, they will conflict.

**Reply by arcline11** [reply]
I discovered this morning I'm a complete idiot. When completing my build, I'd plugged the hub into the SATA cable for power, but forgot the plug the GND FD PMW cable into a SYS_FAN header on the motherboard. I corrected that grievous oversight and since I chose to run all case fans off the hub, so they move in concert, Fan Control now shows them all and the control is Sys Fan #1 as that's where I plugged the PMW cable into. So, stupid mistake rectified.


Fan Control must be open to operate. Thanks for that confirmation.

I use Afterburner sometimes (often actually) to monitor in-game sensors, so I want to run Afterburner and Fan Control both open. I've verified that Fan Control GPU settings override the Afterburner setting. I had Fan Control open, then opened Afterburner and saw the Afterburner Fan Speed % slider jump higher to exactly match the GPU % showing in Fan Control, so far as I can tell since it's trumped by Fan Control, I can run them both at the same time without issue. If I'm wrong on that, I'm open to understanding why.

Many thanks for your kind response. Much appreciated.
